src/isolate.rs

```diff
@@ -190,15 +190,12 @@ impl Isolate {
 
   pub fn respond(&mut self, req_id: i32, buf: Buf) {
     self.state.metrics_op_completed(buf.len());
-
-    // TODO(zero-copy) Use Buf::leak(buf) to leak the heap allocated buf. And
-    // don't do the memcpy in ImportBuf() (in libdeno/binding.cc)
```

Comment:
Looks good - this solves the leak! I think there should be a comment here explaining that memcpy is used.
```
// deno_respond will memcpy the buf into V8's heap, so borrowing a reference here is sufficient. 
```